packages/firebase_analytics/firebase_analytics/example/lib/main.dart

```diff
@@ -6,14 +6,35 @@
 import 'dart:async';
 
 import 'package:firebase_core/firebase_core.dart';
+import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
 import 'package:firebase_analytics/firebase_analytics.dart';
 
 import 'tabs_page.dart';
 
 Future<void> main() async {
   WidgetsFlutterBinding.ensureInitialized();
-  await Firebase.initializeApp();
+  if (kIsWeb) {
```

Comment:
I don't think there's a difference in `initializeApp()` settings.

Reply:
ye but if I don't add the name 'DEFAULT' when running on web without the script initialization it will complain about the default app not initialized, which is temporary until your PR lands in master

Reply:
I thought we have to workaround this before that PR, but I guess I got to change the base?